includes/ApplHandlingByStr.php

```diff
@@ -82,6 +86,22 @@ function showFullStatus( $stat_str )
 }
 
 
+function findPending( $stat_str )
+{
+	$STR = explode ( ';', $stat_str );
+	for ( $i =  1;  $i < count ( $STR ) ; $i++ )
+	{
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->